packages/manager/src/features/PlacementGroups/PlacementGroupsDetail/PlacementGroupsDetail.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+
+import { CircleProgress } from 'src/components/CircleProgress';
+import { DocumentTitleSegment } from 'src/components/DocumentTitle';
+import { ErrorState } from 'src/components/ErrorState/ErrorState';
+import { LandingHeader } from 'src/components/LandingHeader';
+import { NotFound } from 'src/components/NotFound';
+import { SafeTabPanel } from 'src/components/Tabs/SafeTabPanel';
+import { TabLinkList } from 'src/components/Tabs/TabLinkList';
+import { TabPanels } from 'src/components/Tabs/TabPanels';
+import { Tabs } from 'src/components/Tabs/Tabs';
+import { useFlags } from 'src/hooks/useFlags';
+import {
+  useMutatePlacementGroup,
+  usePlacementGroupQuery,
+} from 'src/queries/placementGroups';
+import { getErrorStringOrDefault } from 'src/utilities/errorUtils';
+
+import { getAffinityLabel, getPlacementGroupLinodeCount } from '../utils';
+
+export const PlacementGroupsDetail = () => {
+  const flags = useFlags();
+  const { id, tab } = useParams<{ id: string; tab?: string }>();
+  const history = useHistory();
+  const placementGroupId = Number(id);
+  const {
+    data: placementGroup,
+    error: placementGroupError,
+    isLoading,
+  } = usePlacementGroupQuery(placementGroupId, Boolean(flags.vmPlacement));
+  const {
+    error: updatePlacementGroupError,
+    mutateAsync: updatePlacementGroup,
+    reset,
+  } = useMutatePlacementGroup(placementGroupId);
+  const errorText = getErrorStringOrDefault(updatePlacementGroupError ?? '');
+
+  if (!placementGroup) {
+    return <NotFound />;
+  }
+
+  if (placementGroupError) {
+    return (
+      <ErrorState errorText="There was a problem retrieving your Placement Group. Please try again." />
+    );
+  }
+
+  if (isLoading) {
+    return <CircleProgress />;
+  }
+
+  const linodeCount = getPlacementGroupLinodeCount(placementGroup);
+  const tabs = [
+    {
+      routeName: `/placement-groups/${id}`,
+      title: 'Summary',
+    },
+    {
+      routeName: `/placement-groups/${id}/linodes`,
+      title: `Linodes (${linodeCount})`,
+    },
+  ];
+  const { affinity_type, label } = placementGroup;
+  const affinityLabel = getAffinityLabel(affinity_type);
+  const tabIndex = tab ? tabs.findIndex((t) => t.routeName.endsWith(tab)) : -1;
+
+  const resetEditableLabel = () => {
+    return `${label} (${affinityLabel})`;
+  };
+
+  const handleLabelEdit = (newLabel: string) => {
+    if (updatePlacementGroupError) {
+      reset();
+    }
+
+    return updatePlacementGroup({ label: newLabel });
+  };
+
+  return (
+    <>
+      <DocumentTitleSegment segment={label} />
+      <LandingHeader
+        breadcrumbProps={{
```

Comment:
To remove the hyphen:
![image](https://github.com/linode/manager/assets/122488130/65af46f2-23cc-419f-9c4c-111de9e96b16)

```suggestion
        breadcrumbProps={{
          crumbOverrides: [
            {
              label: 'Placement Groups',
              position: 1,
            },
          ],
```

Reply:
ah yes good call - done as well!